tasks/main.yml

```diff
@@ -79,45 +79,64 @@
                 size: "{{ __hpc_usrlv_size }}"
                 mount_point: "{{ hpc_usrlv_mount }}"
 
-- name: Enable proprietary nvidia-driver
+- name: Update RHUI packages from Microsoft repositories only
   package:
-    name: "{{ hpc_nvidia_driver_module }}"
+    name: "rhui*"
+    state: latest
+    disablerepo: "*"
+    enablerepo: "rhui-microsoft-*"
+  when: hpc_cloud_provider == "azure"
+
+- name: Get current kernel release version
```

Comment:
This is available as `ansible_facts["kernel"]`

Reply:
That's perfect, thanks